runtime/integration-tests/src/pools/pool_starts.rs

```diff
@@ -42,6 +42,22 @@ async fn create_init_and_price() {
 	let loan_amount = 10_000 * DECIMAL_BASE_12;
 	let maturity = 90 * SECONDS_PER_DAY;
 
+	// Get latest block and mutate state
+	env.with_mut_state(Chain::Para(PARA_ID), || {
+		orml_asset_registry::Pallet::<Runtime>::do_register_asset(
+			orml_asset_registry::AssetMetadata {
+				decimals: 18,
+				name: "MOCK TOKEN".as_bytes().to_vec(),
+				symbol: "MOCK".as_bytes().to_vec(),
+				existential_deposit: 0,
+				location: None,
+				additional: common_types::CustomMetadata::default(),
+			},
+			Some(common_types::CurrencyId::AUSD),
+		)
+		.unwrap();
+	});
+
```

Comment:
Didn't you go with the `GenesisConfig`? ^^

I will add this at a later point.

Reply:
This is what holds the PR back. Currently my IDE tells me that `MockGenesisConfigAssetRegistry` doesn't implement `MaybeSerializeDeserialize` and I am in the process of figuring out what this means and how to add it :)

Reply:
Think you can simply derive Encode, Decode. Just a blow in the wind though.